c2cgeoportal/scaffolds/create/project.yaml.in_tmpl

```diff
@@ -1,6 +1,6 @@
 project_folder: {{project}}
 project_package: {{package}}
 host: <host>
-checker_path: /${instance-id}/wsgi/check_collector?
+checker_path: /${instanceid}/wsgi/check_collector?
```

Comment:
This seems to be out of my skills and attributions.
No error in master branch, which has been corrected here https://github.com/camptocamp/c2cgeoportal/commit/c5d3673a5351d001f6b2056c9c0c45d4f56d7b97